search/tests/test_search_result_processor.py

```diff
@@ -264,6 +267,56 @@ def test_excerpt_back(self):
         srp = SearchResultProcessor(test_result, "dog")
         self.assertEqual(srp.excerpt[-33:], "Match upon last word - <b>Dog</b>")
 
+    @ddt.data(
+        (u'"many times"', u"And <b>many times</b> I've cried"),
+        (u'"Don\'t leave"', u"ago <b>Don't leave</b> me waiting here"),
+        (
+            u'"long and winding" leads', 
+            u'The <b>long and winding</b> road That <b>leads</b> to your door'
+        ),
+        (u'"long and winding" night', u'The <b>long and winding</b> road That leads to your door'),
```

Comment:
What are the rules for the excerpt, and why is all of "That leads to your door" included?

Reply:
@robrap Since I am searching for two phrases, "long and winding" and night, I am testing it in two ways. One where the terms can be found close to each other (the test case right above this one) and the second where they are spaced further apart. 

In the first case, the excerpt bolds both phrases since they are close by. In the second it only bolds "long and winding" and doesn't then cut to another excerpt including "night". On second thought though, this doesn't feel like a useful test in this context, so will just remove it.